tests/flow/test_graph_copy_temp_folder.py

```
@@ -0,0 +1,67 @@
+            self.env.skip()
+        self.conn = self.env.getConnection()
+        self.src = 'temp_src_graph'
+        self.dest = 'temp_dest_graph'
+        src_graph = self.db.select_graph(self.src)
+        src_graph.query("RETURN 1")
+
+    def test_01_temp_folder_is_file(self):
+        fd, file_path = tempfile.mkstemp()
+        os.close(fd)
+        try:
+            with pytest.raises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)